lemarche/www/tenders/utils.py

```diff
@@ -97,3 +98,28 @@ def get_or_create_user(request_user, tender_dict: dict, source=user_constants.SO
         if need_to_be_saved:
             user.save()
     return user
+
+
+def duplicate(tender: Tender) -> Tender:
+    FIELDS_TO_REMOVE = (
+        ["_state", "_django_version"]
+        + ["id", "slug", "siae_transactioned", "extra_data", "import_raw_object"]
+        + Tender.READONLY_FIELDS
+    )
```

Comment:
```suggestion
FIELDS_TO_REMOVE = (
        ["_state", "_django_version"]
        + ["id", "slug", "siae_transactioned", "extra_data", "import_raw_object"]
        + Tender.READONLY_FIELDS
    )
def duplicate(tender: Tender, fields_to_remove=FIELDS_TO_REMOVE) -> Tender:
```
Je le mettrais en argument au cas où ^^

L'autre option serait de laisser comme t'as fait initialement et t'ajoute un argument à `duplicate` du type `extra_fields_to_remove`

Reply:
okay t'as raison je vais rendre ca un peu plus réutilisable

Reply:
fixé ici https://github.com/gip-inclusion/le-marche/pull/1097/commits/9661477a0cecf0a64b55f0b29a8e6ef871fb7380